src/main/java/io/codelex/classesandobjects/practice/videostore/Video.java

```diff
@@ -0,0 +1,49 @@
+package io.codelex.classesandobjects.practice.videostore;
+
+public class Video {
+    protected String title;
+    protected boolean flag;
```

Comment:
nedaudz confusing naming, varbūt `isInStock` something like that

Reply:
hey, salabots!